src/components/Blocks/Flickr/Data.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { BlockDataForm } from '@plone/volto/components';
+import { isValidFlickrId } from '../../../helpers';
+import { flickrSchema } from './schema';
+import { useIntl } from 'react-intl';
+
+const FlickrBlockData = (props) => {
+  const { data, block, onChangeBlock } = props;
+  const intl = useIntl();
+  const schema = flickrSchema({ ...props, intl });
+  Object.keys(schema.properties).forEach((key) => {
+    const field = schema.properties[key];
+    const defaultValue = field.default;
+    if (defaultValue !== undefined && data[field] === undefined) {
+      data[field] = defaultValue;
+    }
+  });
```

Comment:
I don't think this part is needed; none of the fields in the schema have a default, do they?